client/app/scripts/superdesk-archive/views/media-box-grid.html

```diff
@@ -47,8 +47,7 @@ <h5 ng-class="{'shifted' : item.highlights}">
 
     <span class="type-icon"><i class="filetype-icon-{{item.type}}"></i></span>
     <span class="urgency-label"><i class="urgency-icon-{{item.urgency}}"></i></span>
-
-    <div class="state-label state-{{item.state}}">{{item.state}}</div>
+    <div class="state-label state-{{item.state}}" ng-class="{state_embargo: item.embargo !== undefined && item.embargo !== null}">{{item.state}}</div>
```

Comment:
Would simply `{state_embargo: item.embargo}` also work? Or do we need to explicitly compare it to both `null` and `undefined`?